### PR TITLE
[SPIKE] [CFP-410] FormBuilder migration - Travel expenses

### PR DESCRIPTION
**Pull request by NenehB**
#### What

Continue the migration away from the deprecated design system to the current GOVUK Frontend.

#### Ticket
[CFP-77-FormBuilder migration - Travel expenses (https://dsdmoj.atlassian.net/browse/CFP-410)

#### Why

- Update the service to use GOV.UK styles, components and patterns
- To improve and resolve the service's accessibility issues
- To reduce code complexity
- To improve maintainability

--------

#### TODO (wip)

- [x]  Update translation
- [x]  Update spec
- [x]  Update aob (link location for lgfs does not work)

### CODE REVIEW

File: app/views/external_users/claims/expenses/_expense_fields.html.haml
Comment (naseberry) [inline app/views/external_users/claims/expenses/_expense_fields.html.haml]: Is this redundant?

File: config/locales/en.yml
Comment (naseberry) [inline config/locales/en.yml]: Nit: should this section be moved to `config/locales/en/models/claim.yml`?

File: features/claims/advocate/scheme_eleven/advocate_fixed_fee_claim_submit.feature
Comment (naseberry) [inline features/claims/advocate/scheme_eleven/advocate_fixed_fee_claim_submit.feature]: Nit: can we remove this step?

File: features/claims/advocate/scheme_nine/advocate_claim_draft_edit_submit.feature
Comment (naseberry) [inline features/claims/advocate/scheme_nine/advocate_claim_draft_edit_submit.feature]: Nit: can we remove this step?

File: app/views/external_users/claims/expenses/_expense_fields.html.haml
Comment (naseberry) [inline app/views/external_users/claims/expenses/_expense_fields.html.haml:61]: Nit: consider adding a `min:` value to  all `govuk_number_field` to limit errors

File: config/locales/en/models/claim.yml
Comment (naseberry) [inline config/locales/en/models/claim.yml]: Did we discuss normalizing this file as part of our formbuilder work?

File: app/validators/base_sub_model_validator.rb
Comment (naseberry) [inline app/validators/base_sub_model_validator.rb]: Nit / Standard: this array should ideally be in alphabetical order

File: app/views/external_users/claims/_error_summary.html.haml
Comment (naseberry) [inline app/views/external_users/claims/_error_summary.html.haml]: Nit / Standard: this array should ideally be in alphabetical order

File: app/views/external_users/claims/expenses/_textarea_additional_info.html.haml
Comment (naseberry) [inline app/views/external_users/claims/expenses/_textarea_additional_info.html.haml]: Nit / Standard: options should be in alphabetical order

File: app/views/external_users/claims/expenses/_expense_fields.html.haml
Comment (naseberry) [inline app/views/external_users/claims/expenses/_expense_fields.html.haml:42]: `f.object.reason_id.eql?(5) ? 'display: block' : 'display: none'` seems to be redundant code.
This ternary would only work on the server side, but this entire step uses client side script to work.

Note: I've created ticket (https://dsdmoj.atlassian.net/browse/CFP-614) to action the  progressive enhancement strategy.

**Review by naseberry — COMMENTED** [review]
Aside from the small nits I've identified, code changes look good.
I've sanity checked changes against a hosted environment, 👍 

I would recommend you get at least 2 approval on this before merging.

**Review by imtiazAhmd — APPROVED** [review]
LGTM 👍🏼

**Review by naseberry — APPROVED** [review]

**Pull request merged**
